### PR TITLE
Add Geometry tile support

### PR DESCRIPTION
**Pull request by bagnell**
Adds Geometry (boxes, cylinders, etc.) tile support. Removes the same features from Vector tiles. Vector tiles are now just points, polygons, and polylines.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@bagnell, thanks for the pull request! Maintainers, we have a signed CLA from @bagnell, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Apps/Sandcastle/gallery/3D Tiles Point Cloud Classification.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
 // For more details, see:
 //    https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/vector-tiles/TileFormats/VectorData
 var classification = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
-    url: 'https://beta.cesium.com/api/assets/3394?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIzZjM4MjljZi0xNTAzLTQ0MzctODhlZi0zOTU3Njg5ODA1Y2QiLCJpZCI6OCwiaWF0IjoxNDgxODI5ODMyfQ.dYlV8_PoXcFNlPHGook5kMzuJMS-Bb9DCMzI1mFVvgE',
+    url: 'https://beta.cesium.com/api/assets/3469?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIzZjM4MjljZi0xNTAzLTQ0MzctODhlZi0zOTU3Njg5ODA1Y2QiLCJpZCI6OCwiaWF0IjoxNDgxODI5ODMyfQ.dYlV8_PoXcFNlPHGook5kMzuJMS-Bb9DCMzI1mFVvgE',
```
Comment (pjcozzi) [inline Apps/Sandcastle/gallery/3D Tiles Point Cloud Classification.html]: Can we add an option to use the geometry tile for rendering vs. classification?

Or is it only suitable for classification right now and we will roadmap rendering, e.g., normals, styling, etc.  I would be fine with that.

File: Source/Scene/Vector3DTileMeshes.js [hunk 2]
```diff
@@ -1,440 +0,0 @@
-define([
```
Comment (pjcozzi) [inline Source/Scene/Vector3DTileMeshes.js]: I think I am missing something - should these files be deleted?  I thought they were used by the b3dm transcode for classification via b3dm.

**Comment by pjcozzi** [comment]
Do you get test failures?  I get failures like:

image

**Comment by pjcozzi** [comment]
Otherwise looks good at quick glance.  Please update the spec too, of course.

File: Apps/Sandcastle/gallery/3D Tiles Point Cloud Classification.html (hunk 1 above)
Comment (bagnell) [reply Apps/Sandcastle/gallery/3D Tiles Point Cloud Classification.html]: This is only suitable for classification right now. I would put geometry tile rendering on the roadmap.

File: Source/Scene/Vector3DTileMeshes.js (hunk 2 above)
Comment (bagnell) [reply Source/Scene/Vector3DTileMeshes.js]: The transcode uses `Vector3DTilePrimitive` which takes buffers and re-batching information.

Comment (bagnell) [reply Source/Scene/Vector3DTileMeshes.js]: I merged in the latest from `b3dm-class` so you can run the Sandcastle examples.

**Comment by bagnell** [comment]
> Do you get test failures?

I don't see any and the Travis tests also passed.

**Comment by bagnell** [comment]
@pjcozzi This should be ready to go.

**Comment by pjcozzi** [comment]
Looks good!

@lilleyse I'm going to merge this (into `b3dm-class`).  Feel free to chime in here next week or later when we have more free time. 😄

**Pull request merged**